tests-hivemq-platform-operator/src/integrationTest/java/com/hivemq/helmcharts/single/CustomOperatorSelectorIT.java

```diff
@@ -10,37 +12,45 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
+@Tag("CustomOperatorConfig")
 @Tag("OperatorSelector")
 class CustomOperatorSelectorIT extends AbstractHelmChartIT {
 
+    private static final @NotNull String PLATFORM_NAME_ALPHA = PLATFORM_RELEASE_NAME + "-alpha";
+    private static final @NotNull String PLATFORM_NAME_BETA = PLATFORM_RELEASE_NAME + "-beta";
+
     @Override
     protected boolean installPlatformOperatorChart() {
         return false;
     }
 
+    @Override
+    protected boolean uninstallPlatformChart() {
+        return false;
+    }
+
+    @AfterEach
+    @Timeout(value = 5, unit = TimeUnit.MINUTES)
+    void tearDown() throws Exception {
+        helmChartContainer.uninstallRelease(PLATFORM_NAME_ALPHA, platformNamespace);
+        helmChartContainer.uninstallRelease(PLATFORM_NAME_BETA, platformNamespace);
```

Comment:
Hmm, that should only be set for the BETA release. Otherwise the operator will still try to reconcile the BETA platform, although the namespace is already being deleted when the ALPHA platform is uninstalled. That will cause confusing error logs.

But anyway good catch, I thought we had an extra delete namespace in the base test, even if the "default" platform helm chart is not uninstalled.